region/brinstar/kraid/Warehouse Energy Tank Room.json

```diff
@@ -88,16 +88,33 @@
           {"and": [
             {"or": [
               "Ice",
-              "canCarefulJump"
+              "canTrickyJump",
+              {"enemyDamage": {
+                "enemy": "Beetom",
+                "type": "contact",
+                "hits": 1
+              }}
             ]},
-            "h_canUseMorphBombs"
+            {"or": [
+              "h_canUseMorphBombs",
+              {"and": [
+                {"resourceAvailable": [ {"type": "Energy", "count": 50} ]},
+                {"or": [
+                  {"ammo": {"type": "Missile", "count": 1}},
+                  {"ammo": {"type": "Super", "count": 1}},
+                  {"ammo": {"type": "PowerBomb", "count": 1}}
+                ]},
+                "h_canUsePowerBombs"
```

Comment:
not really sure why there is a PB required here and in the `OR` just above. It also still feels pretty easy with just ice + PB.

Reply:
Ice+PB is a path through these ORs.  The `enemyDamage` is not a good approximation for beetom damage since you take it every room reset, but the room should be always available as a PB refill.

MorphBombs isn't a trivial way to kill Beetoms and could use more requirements.

The block deals with generating 1 Power Bomb so that you can turn it into full Power Bombs.  Spending 1 PB to gain 1 PB is in line with how the other consumable ammos are giving you a PB.  h_canUsePowerBombs is used to show that the requirement is `canUsePowerBombs`, that you have Morph and Power Bomb.  And it will refill all of your PBs at the end anyway.

Reply:
I see how the `OR` block works now. That's clever. 
I guess I was also just confused on the Energy capacity being needed on the PB strat and not the Bomb strat, but it's so the farm can work without health bomb.